tests/Application/.env

```diff
@@ -31,6 +31,13 @@ JWT_PASSPHRASE=acme_plugin_development
 MAILER_URL=smtp://localhost
 ###< symfony/swiftmailer-bundle ###
 
+###> symfony/mailer ###
+# For Gmail as a transport, use: "gmail://username:password@localhost"
+# For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
+# Delivery is disabled by default via "null://null"
+MAILER_DSN=null://null
+###< symfony/mailer ###
+
```

Comment:
What's this? :man_shrugging:

Reply:
had some problem with swift mailer so decided it would be good to move to symfony mailer anyway

Reply:
as comment at line 37 states # Delivery is disabled by default via "null://null"